config/policy.go

```diff
@@ -22,20 +22,24 @@ type Policy struct {
 	// trusted in order to register.
 	Except []string
 
-	// Trust is either false or true, and determines whether an allowed peer
+	// ExemptRateLimits is either false or true, and determines whether an allowed peer
 	// can skip (true) on-chain verification or not (false), by default.
-	Trust bool
-	// TrustExcept is a list of peer IDs that are exceptions to the trust
+	ExemptRateLimits bool
+	// ExemptRateLimitsExcept is a list of peer IDs that are exceptions to the trust
 	// action.  If Trust is false then all allowed peers must be verified,
 	// except those listed here.  If Trust is true, then only the peers
 	// listed here require verification.
-	TrustExcept []string
+	ExemptRateLimitsExcept []string
+
+	// ExemptSelfPublisher is a list of peer IDs that are allowed to publish on
+	// behalf of other providers.
+	ExemptSelfPublisher []string
```

Comment:
this also seems fine

Reply:
i'd guess we can get away without the full mapping yet, and in fact for e.g the immediate use cases it'll cause additional trouble because the destination providers the publisher will be publishing on behalf of will be dynamic

Reply:
How about this:
```go
    // Publish determines whether or not peers are allowed to publish                                                                                                  
    // advertisements for a provider with adifferen peer ID.                                                                                                           
    Publish bool                                                                                                                                                       
    // PublisherExcept is a list of peer IDs that are exceptions to the Publish                                                                                        
    // policy.  If Publish is false, then all allowed peers cannot publish                                                                                             
    // advertisements for providers with a different peer ID, unless listed in                                                                                         
    // PublishExcept.  If Publish is true, then all allowed peers can publish                                                                                          
    // advertisements for any provider, unless listed in PublishExcept.                                                                                                
    PublishExcept []string                                                                                                                                             
```
For those not allowed by the Publish/PublishExcept policy, there can be a list of specific peer ID they are allowed to publish on behalf of.  That can be added in the future.